src/table.ts

```diff
@@ -512,6 +521,74 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     this.instance.createTable(this.id, options, callback);
   }
 
+  createBackup(
+    id: string,
+    config: CreateBackupConfig
+  ): Promise<CreateBackupResponse>;
+  createBackup(
+    id: string,
+    config: CreateBackupConfig,
+    callback: CreateBackupCallback
+  ): void;
+  createBackup(
+    id: string,
+    config: CreateBackupConfig,
+    callback: CreateBackupCallback
+  ): void;
+  /**
+   * Backup a table with cluster auto selection.
+   *
+   * Backups of tables originate from a specific cluster. This is a helper
+   * around `Cluster.createBackup` that automatically selects the first ready
+   * cluster from which a backup can be performed.
+   *
+   * @param {string} id A unique ID for the backup.
+   * @param {CreateBackupConfig} config Metadata to set on the Backup.
+   * @param {BackupTimestamp} config.expireTime When the backup will be
+   *   automatically deleted.
+   * @param {CallOptions} [config.gaxOptions] Request configuration options,
+   *     outlined here:
+   *     https://googleapis.github.io/gax-nodejs/CallSettings.html.
+   * @param {CreateBackupCallback} [callback] The callback function.
+   * @param {?error} callback.err An error returned while making this request.
+   * @param {Backup} callback.backup The newly created Backup.
+   * @param {Operation} callback.operation An operation object that can be used
+   *     to check the status of the request.
+   * @param {object} callback.apiResponse The full API response.
+   * @return {void | Promise<CreateBackupResponse>}
+   */
+  createBackup(
+    id: string,
+    config: CreateBackupConfig,
+    callback?: CreateBackupCallback
+  ): void | Promise<CreateBackupResponse> {
+    if (!id) {
+      throw new TypeError('An id is required to create a backup.');
+    }
+
+    this.getReplicationStates(config.gaxOptions)
```

Comment:
The mixing of promises and callbacks is hurting my head a little here.  Is it like this because `getReplicationStates` doesn't support callbacks?